src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -302,6 +302,13 @@ void FailSafeCleanup(const chip::DeviceLayer::ChipDeviceEvent * event)
 
     FabricIndex fabricIndex = event->FailSafeTimerExpired.fabricIndex;
 
+    // Report Fabrics table change if SetVIDVerificationStatement had been called.
+    if (event->FailSafeTimerExpired.setVidVerificationStatementHasBeenInvoked)
+    {
+        // Opcreds cluster is always on Endpoint 0.
```

Comment:
```suggestion
        // Opcreds cluster is always on Endpoint 0.
        // Only `Fabrics` attribute is reported since `NOCs` is not reportable (`C` quality).```